pkgs/development/python-modules/hydrogram/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, aiosqlite
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, pyaes
+, pysocks
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "hydrogram";
+  version = "0.1.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "hydrogram";
+    repo = "hydrogram";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-kKzXR8ared2+mHBABxtX+glJ3PMuxA1Pek3DuUkTT40=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    pyaes
+    pysocks
+    aiosqlite
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-asyncio
+  ];
+
+  pythonImportsCheck = [
+    "hydrogram"
+    "hydrogram.errors"
+    "hydrogram.types"
+  ];
+
```

Comment:
Add a update script so r-ryantm can try to update the package. It's easier to maintain later and updates can be deployed much faster.

`passthru.updateScript = gitUpdater { rev-prefix = "v"; };`

should do the trick.